android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -2231,15 +2231,6 @@ public void setFullscreen(boolean fullscreen) {
         updateFullScreenButtonVisbility();
     }
 
-    public void setUseTextureView(boolean useTextureView) {
-        boolean finallyUseTextureView = useTextureView && this.drmUUID == null;
-        exoPlayerView.setUseTextureView(finallyUseTextureView);
-    }
-
-    public void useSecureView(boolean useSecureView) {
-        exoPlayerView.useSecureView(useSecureView);
-    }
```

Comment:
How can we dynamically update view type after initialized?

Reply:
But do it make sens to dynamic update the view type ? I think no ...

Reply:
Yes, I see, that shouldn't be a problem.